beetmoverscript/data/maven_beetmover_task_schema.json

```diff
@@ -0,0 +1,97 @@
+{
+    "title": "Taskcluster beetmover task maven schema",
+    "type": "object",
+    "properties": {
+        "dependencies": {
+            "type": "array",
+            "minItems": 1,
+            "uniqueItems": true,
+            "items": {
+                "type": "string"
+            }
+        },
+        "payload": {
+            "type": "object",
+            "properties": {
+                "upload_date": {
+                    "type": "number"
+                },
+                "build_number": {
+                    "type": "number"
+                },
+                "locale" : {
+                    "type" : "string"
+                },
+                "maxRunTime" : {
```

Comment:
Nit: I just realized `maxRunTime` is never used by beetmoverscript nor scriptworker. It's essentially a param for docker-worker and generic-worker.

Reply:
I agree, although removing this from here means we need to update in-tree for geckoview too + ride the trains. Should we follow-up afterwards in https://github.com/mozilla-releng/beetmoverscript/issues/184 ?

Reply:
Sure!